src/Postmark/Transport.php

```diff
@@ -297,6 +308,22 @@ private function processHeaders(&$payload, $message) {
 		$payload['Headers'] = $headers;
 	}
 
+    /**
+     * Applies the message stream into the API payload.
+     *
+     * @param  array                     $payload
+     * @param  Swift_Mime_SimpleMessage  $message
+     * @return void
+     */
+    private function processMessageStream(&$payload, $message)
+    {
+        if ($message->getHeaders() && $message->getHeaders()->has('X-PM-Message-Stream')) {
```

Comment:
I saw the discussion in https://github.com/wildbit/swiftmailer-postmark/issues/20 about not adding special handling and detecting this in the API instead. However in this csae its necessary, if you have a message stream set as default, then it would be present in the API request, and probably not overwritten by the header that is sent as well.